proto/vtgate.proto

```diff
@@ -147,9 +147,8 @@ message Session {
   // enable_system_settings defines if we can use reserved connections.
   bool enable_system_settings = 23;
 
-  // enable_set_var enables the use of SET_VAR query hint to reduce the number of reserved connection
-  // this feature is part of this RFC (https://github.com/vitessio/vitess/issues/9706)'s second proposal
-  bool enable_set_var = 24;
+  // enable_set_var is not required in session
+  reserved 24;
 }
```

Comment:
Since the original change in #9641 did not go into v13, why do we need to reserve this number? We can delete it.

Reply:
I reserved it for the reason if someone picked up a build from main and so it does not break for them.

Reply:
I am ok to delete this, kindly confirm

Reply:
Should be fine. We don't have to maintain compatibility between day to day builds off main.